### ISSUE TITLE
Training code and The proposed image-text instruction dataset release?

### ISSUE THREAD

**Issue by Amazingren** [hiyamdebary/EarthDial#2]
Dear @hiyamdebary,

Thank you for this amazing work. Recently, I wanted to do some work similarly, and I want to take your method as the baseline, considering its comprehensive coverage for multi-modal input. However,  I just found that the training scripts and the proposed instruction dataset are not yet ready by now.

To this end, may I ask do you have any idea when they might be ready?

Best regards and many thanks,
Bin

**Comment by Amazingren** [comment]
您的来信已经收到，稍后进行回复，谢谢~

**Comment by Amazingren** [comment]
> Hi [@Amazingren](https://github.com/Amazingren)
> 
> Thank you for your interest in our work. We will release the training set in a couple of days.
> 
> Regards Mustansar

Got it. Thank you for the quick reply!

Bests,

**Comment by Amazingren** [comment]
By the way @mustansarfiaz , sorry to bother you again,  regarding the accuracy BigEarthNet (In Tab.5 of your paper) presented in your paper, may I know how the accuracy is calculated?

![Image](https://github.com/user-attachments/assets/bd3cfb26-0439-4c62-a4fb-25068eb63b85)

Since it is a multi-label accuracy dataset, not sure which protocol you adopted in EarthDial?

It would be great and super helpful to hear some instructions regarding this.

Bests

**Comment by Amazingren** [comment]
Never mind, I found it here: 

https://github.com/hiyamdebary/EarthDial/blob/aaa30174c9c1a52767a3f28e33936b31cbdd880e/src/earthdial/eval/rs_classification/eval.py#L10

the recall was used reading the MS classification on BigEarthNet.


Bests
